Circulars - Add Documentation for the Circulars Archive Search Functionality (#2503)
nasa-gcn/gcn.nasa.gov: app/routes/docs.circulars.archive.mdx

  Go to GCN Circulars Archive
</Link>

## Searching the Archive

> You can search the Circulars Archive by keyword. This search tool indexes the subject, body, and submitter of all Circulars. Upon initiating a query, the search tool will return all Circulars that match the query in any of these fields. By default, results will be returned in reverse chronological order, but you can also choose to sort the results by relevance. You can navigate through the search results using pagination controls and specify the number of results to display per page. By default, 100 results are displayed per page.

Reviewer: ```suggestion
You can search the Circulars Archive by keywords or phrases in the subject, body, and submitter fields. By default, results will be returned in reverse chronological order, but you can also choose to sort the results by relevance. You can navigate through the search results using pagination controls and specify the number of results to display per page. By default, 100 results are displayed per page.
```

> Additionally, you can filter search results by the date of their submission. Select the dropdown button labeled "Filter by date". From there, you can either select from a number of predefined time ranges (e.g., "Last Hour", "Last Day", etc.) or specify a custom date range, which can either be set as a start date, an end date, or both. All search filters can be cleared by clicking the 'x' button in the search bar.

Reviewer: ```suggestion
You can also filter search results by submission date. Select the dropdown button labeled "Filter by date". From there, you can either select from a number of predefined time ranges (e.g., "Last Hour", "Last Day", etc.) or specify a custom date range, which can either be set as a start date, an end date, or both. All search filters can be cleared by clicking the 'x' button in the search bar.
```
